src/handle.ml

```diff
@@ -234,6 +234,9 @@ let handle_cmd_aux : sig_state -> command -> sig_state * proof_data option =
         | None    -> fatal t_pos "Cannot infer the type of [%a]." pp t
       in
       out 3 "(eval) %a\n" pp v; (ss, None)
+  | P_why3_init                ->
```

Comment:
Why3 initialization should be automatically done in lambdapi.ml.

Reply:
Yes, @fblanqui is right, you can remove that command. However, initialisation should not be done in `lambdapi.ml` since this file is not loaded by the LSP server. What you should do is call the initialisation code at the initialisation of your module, with something like that (in the main why3-related file):
```ocaml
let _ =
  ... (* Initialization code, run once at load time of the module. *)
```